pyproject.toml

```
@@ -22,8 +22,9 @@ dependencies = [
     "photutils >=1.13.0",
     "pyparsing >=2.4.7",
     "requests >=2.26",
-    "roman_datamodels>=0.25.0,<0.26.0",
+    # "roman_datamodels>=0.25.0,<0.26.0",
     # "roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git",
+    "roman_datamodels @ git+https://github.com/braingram/roman_datamodels.git@rm_ctx",
```

```suggestion
    # "roman_datamodels>=0.25.0,<0.26.0",
    "roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git",
```
As a reminder this needs to be changed to main once https://github.com/spacetelescope/roman_datamodels/pull/514 is merged.